pyspades/server.py

```diff
@@ -338,10 +338,12 @@ def update_entities(self):
             if map_obj.get_solid(entity.x, entity.y, entity.z - 1):
                 moved = True
                 entity.z -= 1
+                # while solid in block above (ie. in the space in which the
+                # entity is sitting), move entity up)
                 while map_obj.get_solid(entity.x, entity.y, entity.z - 1):
                     entity.z -= 1
             else:
-                while not map_obj.get_solid(entity.x, entity.y, entity.z):
+                while map_obj.get_solid(entity.x, entity.y, entity.z) == False:
```

Comment:
`is False` is preferred but that's a nitpick. Might be good to add a comment to explain that this is done since get_solid can return None